posts/2024-01-31-new-way-to-get-a-.gov.md

```diff
@@ -0,0 +1,54 @@
+layout: layouts/post
+date: 2024-01-31
+author: Cameron Dixon
+excerpt: New domain request process for .gov domains
+tags:
+  - posts
+  - Program updates
+---
+
+## U.S.-based government organizations can once again request .gov domains
+
+We’re excited to announce that there is a new way to get a .gov domain. You’re reading this on our new [get.gov](https://get.gov) site. The purpose of this site is to help eligible government organizations get and manage .gov domains. Some highlights:
+
+- [Eligibility for .gov domains](../../domains/eligibility/)
+- [Why you should use .gov](../../domains/benefits/)
+- [How to move to .gov from another domain](../../domains/moving/)
+- [Steps to take before you request a .gov domain](../../domains/before/)
+
+## New process makes it easier to request a .gov
+
+Government organizations at all levels in the U.S. are eligible for .gov domains. This includes cities, towns, counties, election offices, states, tribal governments, courts, federal agencies, and more. 
+
+Our old domain request process required a signature, typically from the senior elected or highest-ranking official in your organization. For many organizations, these requirements led to unnecessary bureaucracy that caused delays. 
+
+**The new request process is fully digital**, doesn’t require a physical signature, and can usually be approved by someone with significant, executive responsibility in your organization, like a senior technology officer or chief administrative official.
+
+<figure style="padding: 5px;">
+<img src="https://github.com/cisagov/get.gov/assets/60157596/d5a66be4-4068-4ebe-bd96-e26677a11005" alt="Gif of the first few screens of the new .gov domain request form" title="Starting the .gov domain request form" style="border: 1px solid;"/> 
+<figcaption style="font-style: italic;">Starting the .gov domain request form</figcaption>
+</figure>
+
+### Now using Login.gov for authentication and identity verification
+
+Login.gov provides a simple and secure process for signing in to many government services with one account. 
+
+We require you to verify your identity with Login.gov **before you can request your first .gov domain**. This is a necessary layer of security that requires you to prove you are you, and not someone pretending to be you. You’ll need a state-issued identification (ID), a Social Security number, and a phone number for identity verification.
+
+- Follow [these steps to create your Login.gov account](https://login.gov/help/get-started/create-your-account/){.usa-link--external}.
+- Read more about [verifying your identity with Login.gov](https://login.gov/help/verify-your-identity/how-to-verify-your-identity/){.usa-link--external}.
+
+## Manage your .gov domain
+
+In November 2023, we launched a new registrar and shared a [summary of the new domain management features](../../about/product/). If you already have a .gov, you can [manage DNS information, domain contacts, and more](../../help/domain-management/). 
```

Comment:
Since we'll be adding a new product update above the November update, should we link to the heading for the Nov 23 update on the /product page instead of the product page generally?

Reply:
Yes. Will do.